### ISSUE TITLE
Snackbar actions

### ISSUE THREAD

**Issue by sgtpepper43** [FezVrasta/snackbarjs#19]
According to the material design spec, snackbars can contain an action (like undo, or retry). An action function and action title should be added to the options, either by adding both of those to the options directly or adding a single action object that contains the title and the callback.

**Comment by murliatdure** [comment]
Do we have some plans around the same?

**Comment by ShashankaNataraj** [comment]
@FezVrasta Can I take this up?

**Comment by FezVrasta** [comment]
Yes guys sorry I'm quite busy. Feel free to send PR, I'll merge them.

**Comment by ShashankaNataraj** [comment]
TY ill take this up

**Comment by gregorhollmig** [comment]
Hi since it is possible to implement actions with html content and onClose event, are you could make a new tag including these changes.
